guides/common/modules/proc_enabling-rh-cloud-and-insights-client-reports-on-hosts.adoc

```diff
@@ -0,0 +1,19 @@
+[id="enabling-rh-cloud-and-insights-client-reports-on-hosts_{context}"]
+= Enabling rh-cloud and insights-client reports on hosts
+
+You can set the `host_registration_insights` parameter to *True* to enable _rh-cloud_ and _insights-client_ reports.
+{Project} will create _rh-cloud_ reports and _insights-client_ will upload reports to the cloud.
+
+Use the following procedure to change the value of `host_registration_insights` parameter.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Host* > *Provisioning Setup* > *Operating Systems*.
+. Select any {RHEL} operating systems for which you want to change the value.
+. On the *Parameters* tab, click on the edit button of `host_registration_insights`.
+. Set the value to *True*.
+
+.Additional resources
+* You can set this parameter at any level.
+For more information, see {ProvisioningDocURL}Host_Parameter_Hierarchy_provisioning[Host parameter hierarchy] in _{ProvisioningDocTitle}_.
```

Comment:
I have been considering adding that module to _Managing hosts_, perhaps as an Appendix, and linking to it locally. I'd like to rewrite the section about registration parameters at some point and link to the appendix from there as well. So it would be worth considering.